nucliadb/src/nucliadb/ingest/service/writer.py

```diff
@@ -136,11 +189,11 @@ async def NewKnowledgeBox(  # type: ignore
                 exc_info=True,
                 extra={"slug": request.slug},
             )
-            return NewKnowledgeBoxResponse(status=KnowledgeBoxResponseStatus.ERROR)
+            return writer_pb2.NewKnowledgeBoxV2Response(status=KnowledgeBoxResponseStatus.ERROR)
 
         else:
             logger.info("KB created successfully", extra={"kbid": kbid})
-            return NewKnowledgeBoxResponse(status=KnowledgeBoxResponseStatus.OK, uuid=kbid)
+            return writer_pb2.NewKnowledgeBoxV2Response(status=KnowledgeBoxResponseStatus.OK)
```

Comment:
you're not passing the kbid anymore to backend after success KB creation. This is needed by backend no?